### PR TITLE
Add view by descending priority for task command.

### PR DESCRIPTION
**Pull request by e0426051**
Minor feature addition. Aims to Address #88 .
Possibility of bugs is low as the command does not use parameters.

### CODE REVIEW

File: src/main/java/seedu/duke/command/CommandSummary.java [hunk 1]
```diff
@@ -27,6 +27,7 @@
     public static final String DURATION = "dur";
     public static final String SELECT = "select";
     public static final String LIST = "list";
+    public static final String PRIORITYVIEW = "priorityview";
 
     /*
```
Comment (gmit22) [inline src/main/java/seedu/duke/command/CommandSummary.java:32]: So what is the command format for this?

File: src/main/java/seedu/duke/command/task/AddTaskCommand.java [hunk 2]
```diff
@@ -47,6 +54,31 @@ public void execute() {
         Ui.showToUserLn("Task successfully created.");
         Ui.showToUserLn(addedTask.toString());
     }
+
+    public boolean doesTitleExist(String title) {
+        boolean titleAlreadyExist = false;
+        if (projectManager.isEmpty()) {
+            return false;
+        }
+        for (int i = 1; i <= projectManager.getSelectedProject().getProjectBacklog().size(); i++) {
```
Comment (gmit22) [inline src/main/java/seedu/duke/command/task/AddTaskCommand.java]: Maybe try iterating through something like `for (Task task: projectmanager.getSelectedProject().getProjectBacklog()` ?

File: src/main/java/seedu/duke/parser/DateTimeParser.java [hunk 3]
```diff
@@ -29,6 +29,47 @@ public static LocalDateTime parseDateTime(String dateTime) throws DukeException
...
+        if (year % 4 == 0) {
+            if (year % 100 != 0) {
+                return true;
+            } else if (year % 400 == 0) {
+                return true;
+            }
+        }
+        return false;
+    }
+
```
Comment (gmit22) [inline src/main/java/seedu/duke/parser/DateTimeParser.java]: What is the purpose of this class?

Comment (e0426051) [reply src/main/java/seedu/duke/parser/DateTimeParser.java]: This is to check February input for leap years, to allow them to enter 29 Feb for those years.

File: src/main/java/seedu/duke/command/task/AddTaskCommand.java (hunk 2 above)
Comment (e0426051) [reply src/main/java/seedu/duke/command/task/AddTaskCommand.java]: Will look into this and implement if possible.

Comment (e0426051) [reply src/main/java/seedu/duke/command/task/AddTaskCommand.java]: Implemented.

**Pull request merged**